src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/WorkTypeProviderTest.java

```diff
@@ -71,8 +72,8 @@ void pactVerificationTestTemplate(PactVerificationContext context) {
     void beforeCreate(PactVerificationContext context) {
         MockMvcTestTarget testTarget = new MockMvcTestTarget();
         testTarget.setControllers(new WorkTypesController(
-            cftWorkTypeDatabaseService,
-            accessControlService
+            accessControlService,
+            workTypesService
```

Comment:
Looks like pact is now broken following update.
https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_WA%2Fwa-task-management-api/detail/PR-361/14/pipeline/185